app/views/schools/year2020/select_cip.erb

```diff
@@ -8,7 +8,7 @@
     <h1 class="govuk-heading-xl">Choose your materials</h1>
     <p class="govuk-body govuk-!-margin-bottom-6">
       All providers cover the same subject matter, but each takes a different approach. Check this
-       <%= govuk_link_to "summary of the support materials (opens in a new tab)", "/pages/year2020-core-materials-info", target: "_blank", rel: "noopener noreferrer" %>
+       <%= govuk_link_to "summary of the support materials (opens in a new tab)", "/pages/year-2020-core-materials-info", target: "_blank", rel: "noopener noreferrer" %>
```

Comment:
FYI, if someone bookmarked the old version and tries to access it, it will result in a sentry error. Not sure if we want to do anything about it, we haven't so far...

Reply:
Yeah, since nobody has been able to see this page on production yet, it won't be an issue. Otherwise we should probably set up 301 redirects when renaming pages